Misc/NEWS.d/next/Core and Builtins/2020-08-31-14-53-17.bpo-41675.VSoqWU.rst

```diff
@@ -0,0 +1,3 @@
+The implementation of :func:`signal.siginterrupt` now uses
+:c:func:`sigaction` instead of the deprecated :c:func:`siginterrupt`. Patch
```

Comment:
You may mention that it's only used when sigaction() is available.

Reply:
Done